### PR TITLE
Reduce check warnings

### PR DESCRIPTION
**Pull request by rjeffman**
Removes a warning on rpmlint and 93 warnings on pytest (`make fastcheck`).

### CODE REVIEW

**Comment by abbra** [comment]
I've restarted Azure checks (known race for DNS tests)

File: ipalib/plugable.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
 you are unfamiliar with this Python feature, see
 http://docs.python.org/ref/sequence-types.html
 """
+from __future__ import unicode_literals
```
Comment (tiran) [inline ipalib/plugable.py]: Master and 4.8 are on Python 3. This future import has no affect in Python 3.

File: ipaserver/dcerpc.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
 # Make sure we only run this module at the server where samba4-python
 # package is installed to avoid issues with unavailable modules
 
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```
Comment (tiran) [inline ipaserver/dcerpc.py]: Same here, please remove

File: ipaserver/dns_data_management.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # Copyright (C) 2016  FreeIPA Contributors see COPYING for license
 #
 
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```
Comment (tiran) [inline ipaserver/dns_data_management.py]: You get the idea... :)

File: ipatests/test_xmlrpc/test_external_members.py [hunk 4]
```diff
@@ -21,6 +21,8 @@
 These tests are skipped if trust is not established.
 """
 
+from __future__ import unicode_literals
```
Comment (tiran) [inline ipatests/test_xmlrpc/test_external_members.py]: and this one

**Comment by rcritten** [comment]
@stanislavlevin reported the failures you're seeing in Travis in https://pagure.io/freeipa/issue/8077 and said he is looking into them.

**Comment by rjeffman** [comment]
This failures @stanislavlevin is looking into are other failures than these ones. I haven't ran with pylint 2.4.0 yet. Will do before going on with this PR.

**Comment by rcritten** [comment]
Can you rebase this? That should resolve the pylint 2.4.0 failures.

**Comment by rjeffman** [comment]
I am asking a new review because I had to change a few places due to pylint 2.4.

**Comment by tiran** [comment]
LGTM!

I have restarted the one failing TravisCI test, too.

**Comment by tiran** [comment]
master:

* bc53544c6f4db00ea11fdf9fa7fb0c421ca63496 Removes rpmlint warning on freeipa.spec.
* c898be1df926bd134ac4d7cd86d10a00ff5cd2ae Removes several pylint warnings.
* 51e0f564870b9b353f0ecf81da700f3d5d1791fb Removed unnecessary imports after code review.
* d4fab33605503fadd1826cd3e879d9c928b8cea9 Removes several pylint warnings.
* 73529e065c901c332caef125f1f12cba33ddaefd Removed unnecessary imports after code review.
* 883b44243ae005c6f2d6d5cab502083f1cc10273 Fixes pylint errors introduced by version 2.4.0.

**Pull request closed without merging**